### DISCUSSION TITLE
ModelSamplingDiscrete/ModelSamplingContinuousEDM/ModelSamplingStableCascade/RescaleCFG，Who knows the actual purpose and parameters of these four nodes？

### DISCUSSION THREAD

**Discussion by liubai-liubai** [Comfy-Org/ComfyUI#3170] [Q&A]
I have been researching the core nodes recently. What are the purposes, usage, and parameters of the four nodes here? Where is the file for reference, and which expert can help?
The four nodes are：ModelSamplingDiscrete/ModelSamplingContinuousEDM/ModelSamplingStableCascade/RescaleCFG

**Comment by kakachiex2** [comment]
1. - :: | Model Sampling ContinuousEDM | Playground2.5 Model | :: { This sampling support model like Playground2.5 }
    - Connection :: ChekpointLoader > Model Sampling ContinuousEDM > KSampler Render.
    
2. - :: | Model Sampling Discreet | LCM Model | LCM Loras | :: { This sampling support LCM models }
    - Connection :: ChekpointLoader > Model Sampling Discreet > KSampler Render.
    
3. - :: | Model Sampling Stable Cascade | Stable Cascade Models | :: { This sampling support stable cascade models }
    - Connection :: ChekpointLoader > Model Sampling StableCascade > KSampler Render.

I dont know about RescaleCFG I will investigate.

**Reply by liubai-liubai** [reply]
Your answer has been very helpful to me. Thank you very much！
